domain/src/main/java/com/globant/domain/util/Constant.kt

```diff
@@ -3,4 +3,5 @@ package com.globant.domain.util
 object Constant {
     const val TYPES_NOT_FOUND = "Pokemon Types Not Found"
     const val SUPERTYPES_NOT_FOUND = "Pokemon Supertypes Not Found"
+    const val SUBTYPES_NOT_FOUND = "Pokemon Subtypes Not Found"
```

Comment:
this constant need to be used in another class or only in PokemonSubtypeDatabaseImpl??. Maybe you can add it in the componion

Reply:
done, thanks